project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clojurewerkz/elastisch "3.0.0-beta2-SNAPSHOT"
+(defproject clojurewerkz/elastisch "5.0.0-alpha1-SNAPSHOT"
```

Comment:
Matching the ES version seemed like the most sensible thing as they're matching versions across the elastic stack now.  Having elasticsch 4.0.0 be compatible with 5.0.0 seems somewhat confusing.

I can revert if you want but it probably needs some discussion.